### PR TITLE
test: added tests for statements methods

### PR DESCRIPTION
**Pull request by yfractal**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Added tests for statements parse_column_default_constraint and sql_column_def_to_grpc_column_def

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

## Refer to a related PR or issue link (optional)
https://github.com/GreptimeTeam/greptimedb/pull/591

### CODE REVIEW

**Review by killme2008 — APPROVED** [review]
LGTM.  Good job.

File: src/sql/src/statements.rs [hunk 1]
```diff
@@ -531,4 +535,61 @@ mod tests {
...
+            data_type: DataType::Double,
+            collation: None,
+            options: vec![],
+        };
+
+        let grpc_column_def = sql_column_def_to_grpc_column_def(column_def).unwrap();
+
+        assert_eq!("col", grpc_column_def.name);
+        assert!(grpc_column_def.is_nullable); // nullable when options are empty
+        assert_eq!(10, grpc_column_def.datatype);
```
Comment (waynexia) [inline src/sql/src/statements.rs:578]: Better to use `ColumnDataType::Float64 as i64`

Comment (yfractal) [reply src/sql/src/statements.rs]: thanks~ updated @waynexia 

And I removed hardcode number in src/datanode/src/server/grpc/ddl.rs too, do you think I should create another PR  for ddl?

Comment (waynexia) [reply src/sql/src/statements.rs]: Well done! It's ok to include this change 🥰

**Review by waynexia — APPROVED** [review]

**Pull request merged**